tests/e2e/device-plugin_test.go

```diff
@@ -63,6 +63,7 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 				fmt.Sprintf("image.repository=%s", *ImageRepo),
 				fmt.Sprintf("image.tag=%s", *ImageTag),
 				fmt.Sprintf("image.pullPolicy=%s", *ImagePullPolicy),
+				"runtimeClassName=nvidia",
```

Comment:
Is there a reason why to use this? (without the test was working)

Reply:
I am not against, but if we add this, we need to deploy the `RuntimeClass` CR object, or the test won't run

Reply:
It means that the underlying container runtime doesn't have to be configured with nvidia as the default runtime.